### ISSUE TITLE
bower ENOTDIR error

### ISSUE THREAD

**Issue by SeyZ** [tonycoco/heroku-buildpack-ember-cli#109]
Hey @tonycoco, here's my error when deploying your buildpack on Heroku (worked before):

```
seyz:client/ (master) $ git push heroku master -f                                                                                                                                                                                                                  [18:04:58]
Warning: Permanently added 'heroku.com,50.19.85.132' (RSA) to the list of known hosts.
Counting objects: 297, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (283/283), done.
Writing objects: 100% (297/297), 723.58 KiB | 0 bytes/s, done.
Total 297 (delta 49), reused 0 (delta 0)
remote: Compressing source files... done.
remote: Building source:
remote: 
remote: -----> Fetching custom git buildpack... done
remote: -----> Ember CLI app detected
remote: -----> Exporting config vars to environment
remote: -----> Building environment: production
remote: -----> Requested node range:  0.12.2
remote: -----> Resolved node version: 0.12.2
remote: -----> Downloading and installing node
remote: -----> Using default npm version: 2.7.4
remote: -----> Downloading and installing nginx
remote: -----> Adding boot script
remote: -----> Copying configs
remote: -----> Forcing rebuild of all node_modules. Pruning all node_modules.
remote: -----> Forcing re-installation of all bower_components. Pruning all bower_components.
remote: -----> Creating node_modules cache directory
remote: -----> Creating bower_components cache directory
remote: -----> Installing bower which is required by other dependencies
remote:        bower@1.6.2 node_modules/bower
remote: -----> Pruning cached dependencies not specified in package.json
remote: -----> Pruning cached bower dependencies not specified in bower.json
remote:        bower ENOTDIR       ENOTDIR, not a directory '/tmp/build_37e2b77f1b99707bfca62c125116fd48/bower_components'
remote: 
remote:  !     Push rejected, failed to compile Ember CLI app
remote: 
remote: Verifying deploy....
```

Any idea of `remote:        bower ENOTDIR       ENOTDIR, not a directory '/tmp/build_37e2b77f1b99707bfca62c125116fd48/bower_components'` means ?

**Comment by danconnell** [comment]
I get the same error. Isn't resolved by purging the cache and using rebuild_all

**Comment by danconnell** [comment]
I don't understand what broke since it was working less than an hour ago and now it's not.

If anyone has a workaround, I'd appreciate it.

**Comment by lucascampbell** [comment]
broken for me as well.

**Comment by alexgb** [comment]
Yep, we're also experiencing the same issue starting today

**Comment by aleontiev** [comment]
Broken here, REBUILD_ALL does not work, destroying and recreating the app does not work :(

Blocking a release for us today

**Comment by aleontiev** [comment]
Seems to be caused by `bower@1.6.2` -- forking this buildpack and forcing an install of the last known good version (`bower@1.5.3`) did the trick for us.

**Comment by aleontiev** [comment]
See this PR: https://github.com/AltSchool/heroku-buildpack-ember-cli/pull/1/files

Would be nice if this buildpack could support a way to force a specific version of bower.

**Comment by tonycoco** [comment]
Could you PR that over to this repo, @aleontiev?

**Comment by tonycoco** [comment]
Never mind, @aleontiev. Got merged with #110
